crates/types/src/deployment.rs

```diff
@@ -78,6 +80,36 @@ impl schemars::JsonSchema for DeploymentId {
     }
 }
 
+/// Deployment which was chosen to run an invocation on.
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+pub struct ChosenDeployment {
```

Comment:
Maybe `InvocationPinnedDeployment`?

Reply:
Also probably belongs to invocation types.

Reply:
Hmm both names are not great tbh. Maybe only `PinnedDeployment`.